src/main/java/guru/qa/allure/notifications/clients/rocket/RocketChatClient.java

```diff
@@ -0,0 +1,49 @@
+package guru.qa.allure.notifications.clients.rocket;
+
+import guru.qa.allure.notifications.clients.Notifier;
+import guru.qa.allure.notifications.config.rocket.RocketChat;
+import guru.qa.allure.notifications.exceptions.MessagingException;
+import guru.qa.allure.notifications.template.RocketTemplate;
+import guru.qa.allure.notifications.template.data.MessageData;
+import java.io.ByteArrayInputStream;
+import java.util.HashMap;
+import java.util.Map;
+import kong.unirest.ContentType;
+import kong.unirest.Unirest;
+
+public class RocketChatClient implements Notifier {
+
+    private final Map<String, Object> body = new HashMap<>();
```

Comment:
it should be a variable in method `sendText`

Reply:
ok, fixed